sktime/classification/tests/test_all_classifiers.py

```diff
@@ -86,13 +86,20 @@ def test_classifier_output(self, estimator_instance, scenario):
 
         # check predict
         assert isinstance(y_pred, np.ndarray)
-        assert y_pred.shape == (X_new_instances,)
+        assert y_pred.shape == (X_new_instances,) or y_pred.shape == (
+            X_new_instances,
+            y_pred.ndim,
+        )
         assert np.all(np.isin(np.unique(y_pred), np.unique(y_train)))
 
         # check predict proba (all classifiers have predict_proba by default)
         y_proba = scenario.run(estimator_instance, method_sequence=["predict_proba"])
         assert isinstance(y_proba, np.ndarray)
-        assert y_proba.shape == (X_new_instances, n_classes)
+        assert y_proba.shape == (X_new_instances, n_classes) or y_proba.shape == (
+            X_new_instances,
```

Comment:
is this the sklearn convention?

Reply:
I am not sure about that. I just modified the check based on the previous one, except that now it also checks for multioutput.